### PR TITLE
peco 0.3.2

### PR DESCRIPTION
**Pull request by kozo2**
bump

### CODE REVIEW

File: Library/Formula/peco.rb [hunk 1]
```diff
@@ -34,7 +34,7 @@ class Peco < Formula
 
   go_resource "github.com/peco/peco" do
     url "https://github.com/peco/peco.git",
-      :revision => "700b77b5ba57ce0cc57339d063a24bb06a485eca"
+      :revision => "d36c492e5032da478fae3a05744cfc5cf9c75759"
   end
```
Comment (mikemcquaid) [inline Library/Formula/peco.rb]: Seems weird that we're effectively downloading the same formula twice?

**Comment by kozo2** [comment]
@mikemcquaid I removed go_resource for peco

File: Library/Formula/peco.rb [hunk 2]
```diff
@@ -32,15 +36,10 @@ class Peco < Formula
...
-    url "https://github.com/peco/peco.git",
-      :revision => "700b77b5ba57ce0cc57339d063a24bb06a485eca"
-  end
-
-  depends_on "go" => :build
-
   def install
     ENV["GOPATH"] = buildpath
+    mkdir_p buildpath/"src/github.com/peco"
+    ln_s buildpath, buildpath/"src/github.com/peco/peco"
```
Comment (mikemcquaid) [inline Library/Formula/peco.rb]: :+1:

**Comment by asparagui** [comment]
Thanks!

**Pull request closed without merging**